core/services/chainlink/config_general.go

```diff
@@ -166,6 +167,26 @@ func (g *generalConfig) AllowOrigins() string {
 	return *g.c.WebServer.AllowOrigins
 }
 
+func (g *generalConfig) AuditLoggerEnabled() bool {
+	return *g.c.AuditLogger.Enabled
```

Comment:
On second thought, this explicit `Enabled *bool` field may not be strictly necessary. :thinking:  I think it would be ok to drop the _field_, but keep the _method_ and implement based on :
```suggestion
	return g.c.AuditLogger != nil
```
But really nbd in for this PR, as long as it is internally consistent. We haven't sorted this out as a whole for new config yet, but will be working on it very soon, and it will be easy to tweak this if necessary.